include/aws/http/private/h2_connection.h

```diff
@@ -146,6 +147,8 @@ enum aws_h2_data_encode_status {
 
 /* When window size is too small to fit the possible padding into it, we stop sending data and wait for WINDOW_UPDATE */
 #define AWS_H2_MIN_WINDOW_SIZE (256)
+/* Default value for thread_data.max_closed_stream_cache_size */
+#define AWS_H2_DEFAULT_MAX_CACHE_SIZE (32)
```

Comment:
naming suggestion: AWS_H2_DEFAULT_MAX_CLOSED_STREAMS
"cache" is pretty vague unless you know the code

also, the comment refers to a variable that doesn't exist anymore